siliconcompiler/core.py

```diff
@@ -537,6 +551,10 @@ def create_cmdline(self,
 
             # Cycle through all items
             for item in vals:
+                if item is None:
```

Comment:
(Why) Is `item = ''` preferred over `item = None`?

Reply:
because later we blindly call item.split() and if that is a `None` it raises an exception

Reply:
Ok I guess.